src/Azure.Functions.Cli/Helpers/ProgrammingModelHelper.cs

```diff
@@ -15,12 +15,15 @@ public static IEnumerable<ProgrammingModel> GetProgrammingModels()
         // Given a worker runtime, this function returns a collection of the programming models that are supported.
         public static IEnumerable<ProgrammingModel> GetSupportedProgrammingModels(WorkerRuntime workerRuntime)
         {
-            var allProgrammingModels = GetProgrammingModels();
-            if (workerRuntime != WorkerRuntime.python)
+            switch (workerRuntime)
             {
-                return allProgrammingModels.Where(pm => pm != ProgrammingModel.V2);
+                case WorkerRuntime.python:
+                    return new List<ProgrammingModel>() { ProgrammingModel.V1, ProgrammingModel.V2 };
+                case WorkerRuntime.node:
+                    return new List<ProgrammingModel>() { ProgrammingModel.V3, ProgrammingModel.V4 };
```

Comment:
So we don't have v1 and v2 in Node? Looks like we have gone through multiple models for node.

Reply:
v1 and v2 of the node programming model were associated with host runtime v2 and v3 which are out of support by now